### PR TITLE
[FLINK-6823] Activate checkstyle for runtime/broadcast

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-runtime/src/main/java/org/apache/flink/runtime/broadcast/BroadcastVariableManager.java [hunk 1]
```diff
@@ -18,34 +18,42 @@
...
 import org.apache.flink.runtime.operators.BatchTask;
 
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * The BroadcastVariableManager is used to manage the materialization of broadcast variables. References to materialized
+ * broadcast variables are cached and shared between parallel subtasks. A reference count is maintained to track whether
+ * a the materialization may be cleaned up.
```
Comment (greghogan) [inline flink-runtime/src/main/java/org/apache/flink/runtime/broadcast/BroadcastVariableManager.java]: Delete leading `a`.

File: flink-runtime/src/main/java/org/apache/flink/runtime/broadcast/BroadcastVariableManager.java [hunk 2]
```diff
@@ -18,34 +18,42 @@
...
+
 	private final ConcurrentHashMap<BroadcastVariableKey, BroadcastVariableMaterialization<?, ?>> variables =
 							new ConcurrentHashMap<BroadcastVariableKey, BroadcastVariableMaterialization<?, ?>>(16);
-	
+
 	// --------------------------------------------------------------------------------------------
-	
+
+	/**
+	 * Materializes the broadcast variable for the given name, scoped to the given task and it's iteration superstep. An
```
Comment (greghogan) [inline flink-runtime/src/main/java/org/apache/flink/runtime/broadcast/BroadcastVariableManager.java]: `it's` -> `its`

**Comment by zentol** [comment]
@greghogan fixed.

**Comment by greghogan** [comment]
+1

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**